yapapi/executor/strategy.py

```diff
@@ -85,27 +87,30 @@ async def score_offer(self, offer: rest.market.OfferProposal) -> float:
         """Score `offer` according to cost for expected computation time."""
 
         linear: com.ComLinear = com.ComLinear.from_properties(offer.props)
-
+        self._logger.debug("Scoring %s: %s", offer.id, linear)
         if linear.scheme != com.BillingScheme.PAYU:
+            self._logger.debug("Scoring %s: unsupported scheme '%s'", offer.id, linear.scheme)
```

Comment:
```suggestion
            self._logger.debug("Rejecting offer %s: unsupported scheme '%s'", offer.id, linear.scheme)
```

Reply:
All suggestions taken into account in https://github.com/golemfactory/yapapi/pull/216/commits/0a93512dbf218d162daf45fd3b7a84632e32ed37